components/notification-bar/notification-bar.scss

```diff
@@ -0,0 +1,7 @@
+.notification-bar {
+  @extend .open-sans;
+
+  font-weight: 600;
+  background: rgba($pulse-purple, 0.2);
+  padding: 1rem;
```

Comment:
Is there a Bootstrap padding class that can be used instead?

Reply:
you mean do `@extend .p-3` here or add `p-3` as class to the notification component?

Reply:
Just adding a class to the component...

Reply:
@gvn this component will be used across the site. I wanna make sure it has consistent/fixed padding so I'd like keeping the padding rule here.

Reply:
If the class is in the React component definition it should be consistent everywhere it's used.